README.md

```diff
@@ -1 +1,92 @@
-# Rye
\ No newline at end of file
+# Rye
+
+[![Carthage Compatible](https://img.shields.io/badge/carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage)
+![Plaforms](https://img.shields.io/badge/platforms-iOS%20-lightgrey.svg)
+[![GitHub license](https://img.shields.io/badge/license-MIT-blue.svg)](https://github.com/nodes-ios/Rye/blob/master/LICENSE)
+[![CircleCI](https://circleci.com/gh/nodes-ios/Rye.svg?style=shield)](https://circleci.com/gh/nodes-ios/Rye)
+
+## Intro
+
+Rye will enable you to display non intrusive 'Toast' type alerts to your users. 
+You can choose to display the default Rye alert type or go fully custom and display your own `UIView`.
+
+## 📝 Requirements
+
+iOS 12
+Swift 5
+
+## 📦 Installation
+
+### Carthage
+~~~bash
+github "nodes-ios/Rye"
+~~~
+
+## 💻 Usage
+
+```swift 
+import Rye
+```
+
+### Display Default Rye
+
+```swift 
+
+let ryeConfiguration: RyeConfiguration = [Rye.Configuration.Key.text: "Message for the user"]
+
+let rye = RyeViewController.init(type: .`default`(configuration: ryeConfiguration), timeAlive: 2)
```

Comment:
Does this have to be \`default\`? AFAIK that's only needed when you define that enum case.

Reply:
i can remove the ``around it, but couldn't come with a better naming, so I am open to suggestions

Reply:
.standard?

Reply:
that can work

Reply:
I mean, it should work fine with just .default :) It's only in the Enum definition where you need to use this AFAIK.